specs/Spec.SPARKLE.fst

```diff
@@ -355,7 +353,7 @@ let sparkle512 steps input =
       let b6 = arx rcon.[6] b6 in
       let b7 = arx rcon.[7] b7 in
       l8 (b0,b1,b2,b3,b4,b5,b6,b7)
-  ) (b0,b1,b2,b3,b4,b5,b7,b7)
```

Comment:
Heads up @beurdouche  @karthikbhargavan  @polubelova if you're relying on this specification for anything.

Reply:
Nice catch, Thanks Santiago! I have a plan to implement lightweight contestant and add tests later in the summer.